src/helpers.c

```diff
@@ -584,14 +584,14 @@ enum swupd_code swupd_init(enum swupd_init_config config)
 			ret = SWUPD_LOCK_FILE_FAILED;
 			goto out_fds;
 		}
+	}
```

Comment:
So signature_init now needs to run even as a non-root. Because without signature_init here, signature_verify during get_latest_version will complain saying it cannot find local issuer certificate. 

for eg: swupd check-update can run by a normal user. For such a user,  swupd check-update run with SWUPD_NO_ROOT which then wouldn't perform signature_init and cause what above problem.

let me know if that explains.

Reply:
ok